### PR TITLE
add open space page 

### PR DESCRIPTION
**Pull request by MadRex2000**
(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [x] **Other (please describe)**

## Description
**Describe what the change is**
solved issue #843 
Add open space page to PyCon TW 2020 website and update the open space page's content.

## Steps to Test This Pull Request
Go to ``localhost:8000/<lang>/events/open-spaces/`` by your web browser

### CODE REVIEW

File: src/templates/pycontw-2020/contents/en/events/open-spaces.html [hunk 1]
```diff
@@ -56,4 +57,4 @@ <h2>Ideas for Open Spaces</h2>
 
 <p>Take a look at what Open Spaces are there at <a href="https://us.pycon.org/2019/events/open-spaces" target="_blank" rel="noopener">PyCon US 2019</a>. There are some great examples for Open Space topics</p>
```
Comment (yychen) [inline src/templates/pycontw-2020/contents/en/events/open-spaces.html:58]: 我看原本的版本沒有這個東西，想問問看 @Lee-W 有沒有要留著。(中文的也一樣)

File: src/templates/pycontw-2020/_includes/menu.html [hunk 2]
```diff
@@ -90,9 +90,7 @@
 				<li><a href="{{ events_talk_list_url }}">{% trans 'Talks' %}</a></li>
 				<li><a href="{{ events_tutorial_list_url }}">{% trans 'Tutorials' %}</a></li>
 				<li><a href="{{ events_sprint_url }}">{% trans 'Sprints' %}</a></li>
-				<!--{% comment %}-->
 				<li><a href="{{ events_openspaces_url }}">{% trans 'Open Spaces' %}</a></li>
```
Comment (yychen) [inline src/templates/pycontw-2020/_includes/menu.html:93]: 可以跟 Sprints 對調嗎? (吹毛求疵，希望可以跟 Overview 上面的順序相符合) 感恩!

File: src/templates/pycontw-2020/contents/en/events/open-spaces.html (hunk 1 above)
Comment (yychen) [reply src/templates/pycontw-2020/contents/en/events/open-spaces.html:58]: PS. 參考頁面: https://staging.pycon.tw/2020/zh-hant/events/open-spaces/

File: src/templates/pycontw-2020/_includes/menu.html (hunk 2 above)
Comment (MadRex2000) [reply src/templates/pycontw-2020/_includes/menu.html:92]: OK 已對調

File: src/templates/pycontw-2020/contents/en/events/open-spaces.html (hunk 1 above)
Comment (Lee-W) [reply src/templates/pycontw-2020/contents/en/events/open-spaces.html:58]: hmmm... link 到其他 PyCon 好像總覺得有點怪怪的xD 先移掉好了？

Comment (MadRex2000) [reply src/templates/pycontw-2020/contents/en/events/open-spaces.html:58]: 前幾年 open space 的頁面都會 link 到當年的 PyCon US 給想參與的人做參考，可以拿掉但今年 PyCon US 好像沒有 open space 所以沒有頁面

Comment (yychen) [reply src/templates/pycontw-2020/contents/en/events/open-spaces.html:58]: 我個人是覺得移掉 +1 XD

Comment (MadRex2000) [reply src/templates/pycontw-2020/contents/en/events/open-spaces.html]: 那就移掉吧

**Pull request merged**